thinc/util.py

```diff
@@ -163,7 +161,7 @@ def prefer_gpu(gpu_id: int = 0) -> bool:  # pragma: no cover
 def require_gpu(gpu_id: int = 0) -> bool:  # pragma: no cover
     from .backends import set_current_ops, CupyOps
 
-    if not gpu_is_available():
+    if not has_cupy_gpu:
         raise ValueError("No GPU devices detected")
```

Comment:
For now perhaps we want to make this error more specific to say "No (CUDA) GPU devices detected" ?

Reply:
Fixed, though I didn't use parentheses.